helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -18,6 +18,11 @@ receivers:
   k8s_cluster:
     auth_type: serviceAccount
     metadata_exporters: [signalfx]
+  {{- if .Values.otelK8sClusterReceiver.k8sEventsEnabled }}
+  smartagent/kubernetes-cluster:
```

Comment:
Why `smartagent/kubernetes-cluster`? We already collect k8s metrics with otel native k8scluster receiver.  you need to enable `smartagent/kubernetes-events` monitor instead

Reply:
In both monitors `smartagent/kubernetes-cluster` and `smartagent/kubernetes-events` we are facing same issue so i have added that one, but as you suggested I have updated it to `smartagent/kubernetes-events` monitor.